spotinst/azure_v3/stateful_node_azure_image/fields_spotinst_stateful_node_azure_image.go

```diff
@@ -297,6 +301,9 @@ func expandStatefulNodeAzureGallery(data interface{}) (*azure.Gallery, error) {
 			if v, ok := m[string(VersionName)].(string); ok && v != "" {
 				gallery.SetVersionName(spotinst.String(v))
 			}
```

Comment:
SpotAccountID not added into flatten function

Reply:
Added to flatten func.